### ISSUE TITLE
18 false positive domain domains in HaGeZi Multi Pro++

### ISSUE THREAD

**Issue by GidCat** [hagezi/dns-blocklists#4148]
### Which AdBlocker/DNS cloud service do you use?

NextDNS

### ControlD users

- [X] IMPORTANT - I assure that I have not set the Block Response in ControlD to Custom or Branded and can confirm that the problem still occurs.

### NextDNS users

- [X] IMPORTANT - I can assure that I disabled the block page in NextDNS and can confirm that the problem still occurs.

### With which block list(s) does the problem occur?

Multi PRO++

### Which domain(s) should be unblocked?

careers.marriott.com
jobs.marriott.com
click.dice.com
careers.wholefoodsmarket.com
grayjay.app
play.google.com
linksmail.geosolinc.com
app-analytics-v2.snapchat.com
tr.snapchat.com
ad.doubleclick.net
vvid30c.site
community.frame.work
content.analytics.insperity.com
content.usps.com
vincheck.info
tracking.icims.com
once.deputy.com
guardianproject.info

### Why should the domain(s) be unblocked?

A few of the respective websites do not load to the point where they are functional but most simply do not load at all with these domains blocked.

### Privacy

- [X] I confirm that the report does not contain any private information.

**Comment by hagezi** [comment]
@GidCat 

**Not blocked in my lists** - What is blocking this for you?:
```
click.dice.com
grayjay.app
play.google.com
vvid30c.site
community.frame.work
content.usps.com
vincheck.info
tracking.icims.com
once.deputy.com
guardianproject.info
```

**Blocked by CNAME** `sites.paradox.ai`, will be removed:
```
careers.marriott.com
jobs.marriott.com
careers.wholefoodsmarket.com
```


**Blocked by CNAME** `tracking.socketlabs.com`, referral domain, will be removed:
`linksmail.geosolinc.com`

**Remain blocked:**
`ad.doubleclick.net` - link tracker, which is also used for non-link tracking, only blocked in the aggressive lists, see here: https://github.com/hagezi/dns-blocklists/wiki/FAQ#referral

`app-analytics-v2.snapchat.com` / `tr.snapchat.com` - Snapchat Analytics & Tracking. So far no restrictions due to blocking. What exactly is not working?

**Details are missing, what is not working due to the blocking?**
`content.analytics.insperity.com`

**Comment by jordansworld** [comment]
i have been using snapchat with those filters blocked and everything works completely fine

**Comment by GidCat** [comment]
`These are domains that I've needed to allow over the past few years. I backtracked and tried to find what the reason for allowing was. Here are my findings from what Next DNS' logs say:   click.dice.com - redirects to thirdparty.bnc.lt - Blocked by disguised third party trackers - Dice.com loads without it, may not need to be allowed.`
`grayjay.app - Is no longer being blocked, unsure of what blocked it originally.`

`play.google.com - Blocked by categories.gaming (not HaGeZi).`
`vvid30c.site - Is no longer being blocked, unsure of what blocked it originally.`

`community.frame.work - blocked by .work (TLD)`

`content.usps.com - Bad request (maybe I needed to follow a specific link?)`

`vincheck.info - Blocked by .info (TLD)`

`tracking.icims.com - Is no longer being blocked, unsure of what blocked it originally.`

`once.deputy.com - Is no longer being blocked, unsure of what blocked it originally.`

`guardianproject.info - Blocked by .info (TLD)`

`content.analytics.insperity.com - I believe that a section of the website was not loading with this being blocked but can't find which one it was. This can be disregarded.  It seems that the top level domain blocks aren't due to the HaGeZi list, I apologize for the confusion. Next time I find false positives, I will now know how to discern if they're being blocked by HaGeZi or not and share detailed notes on what the breakage is. Thank you and have a wonderful day!`
